AndroidNativeKotlinTemplate/gradle.properties

```diff
@@ -1,6 +1,6 @@
 cdvCompileSdkVersion=android-33
 cdvMinSdkVersion=24
 android.useAndroidX=true
-android.enableJetifier=true
+android.enableJetifier=false
```

Comment:
Why the change?

Reply:
The build analyzer suggested it with a confirmation that there are no libraries in the dependency graph which require it.  I've used this quite a bit so I have some trust in the tool, but wouldn't mind reverting it, either.  The build time gain is probably negligible.

Reply:
👍